spec/lib/sprig_spec.rb

```diff
@@ -7,6 +7,30 @@
     Sprig::Configuration.stub(:new).and_return(configuration)
   end
 
+  describe '#adapter' do
+    let(:supported_adapters) do
+      [:active_record, :mongoid]
+    end
+
+    it { expect(described_class).to respond_to(:adapter).with(0).arguments }
+
+    it { expect(supported_adapters).to include described_class.adapter }
```

Comment:
Mostly that the default `.adapter` method returns *something*. Looking at it again, a simple not-nil check would be clearer. Will fix.